frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -59,12 +60,14 @@ OmegaUp.on('ready', () => {
           activeTab,
         },
         on: {
-          'navigate-to-problem': (source: ActiveProblem) => {
-            const problemIndex = problemsStore.state.index[source.alias];
-            if (typeof problemIndex !== 'undefined') {
+          'navigate-to-problem': (request: ActiveProblem) => {
+            const problemAliasStored = problemsStore.state.sortedAliases.find(
+              (alias: string) => alias === request.problem.alias,
+            );
+            if (typeof problemAliasStored !== 'undefined') {
               contestPractice.problemInfo =
-                problemsStore.state.problems[problemIndex];
-              window.location.hash = `#problems/${source.alias}`;
+                problemsStore.state.problems[problemAliasStored];
```

Comment:
```suggestion
            if (Object.prototype.hasOwnProperty.call(problemsStore.state.problems, request.problem.alias)) {
              contestPractice.problemInfo =
                problemsStore.state.problems[request.problem.alias];
```

aquí ya sabemos el alias y lo único que nos interesa es si está presente en el mappeo o no.

Reply:
Listo